### DISCUSSION TITLE
Ban in discord server

### DISCUSSION THREAD

**Discussion by alvaroartano** [Rapptz/discord.py#6437] [General] [answered]
Hi guys! I was banned a long time ago and now I started coding another time, can someone please unban me from the guild?

alvaro.py#1051

Thanks guys

**Comment by Rapptz** [comment, marked as answer]
You've been unbanned, but please don't use GitHub for this. It's not the proper place.

**Comment by alvaroartano** [comment]
Sure thanks a lot

El vie, 19 feb 2021 a las 7:52, Danny (<notifications@github.com>) escribió:

> You've been unbanned, but please don't use GitHub for this. It's not the
> proper place.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Rapptz/discord.py/discussions/6437#discussioncomment-383251>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG3CQ2TV4NYV4TEPB6YMUR3S7YDD7ANCNFSM4X22YW4Q>
> .
>


-- 
Alvaro Artano
